src/components/enterprise/dashboard/DashboardPage.jsx

```diff
@@ -9,17 +9,20 @@ import { DashboardSidebar } from './sidebar';
 
 const DashboardPage = (props) => {
   const { pageContext } = props;
-  const { enterpriseName } = pageContext;
+  const { enterpriseName, enterpriseEmail } = pageContext;
   return (
-    <Layout pageContext={props.pageContext}>
+    <Layout pageContext={pageContext}>
       <Hero title={enterpriseName} />
       <div className="container py-5">
         <div className="row">
           <MainContent>
             <DashboardMainContent />
           </MainContent>
           <Sidebar>
-            <DashboardSidebar />
+            <DashboardSidebar
+              enterpriseName={enterpriseName}
```

Comment:
Instead of passing these props into `DashboardSidebar`, that component can pull from `LayoutContext` the `enterpriseName` and `enterpriseEmail`.

```
class DashboardSidebar extends React.Component {
  static contextType = LayoutContext;
  render() {
    const { pageContext: { enterpriseName, enterpriseEmail } } = this.context;
    // ...
  }
}
```

Reply:
Cool. Yeah, I thought about this when you demoed your PRs yesterday. I will update this.